frontend/src/app/modules/user-profile/user-details/user-details/user-details.component.ts

```diff
@@ -102,15 +125,29 @@ export class UserDetailsComponent extends BaseComponent implements OnInit {
         return this.detailsForm.get('email') as FormControl;
     }
 
-    get instagram(): FormControl {
-        return this.detailsForm.get('instagram') as FormControl;
+    selectInterest($event: Event, i: number) {
+        const ev = $event.target as HTMLInputElement;
+
+        // if (ev.checked) {
+        //     this.selectedTags = this.selectedTags.concat({ name: this.tagsList[i].icon_name });
+        // } else {
+        //     this.selectedTags = this.selectedTags.filter(x => x.name !== this.tagsList[i].icon_name);
+        // }
+
+        this.selectedTags = (ev.checked)
+            ? this.selectedTags.concat({ name: this.tagsList[i].icon_name })
```

Comment:
concat -> [...this.selectedTags, {name...}]

Reply:
Sorry,  didn't get what you mean. Concat is not good?